tests/e2e/pact/specs/eps-sandbox.spec.ts

```diff
@@ -17,20 +17,20 @@ jestpact.pactWith(
   /* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
Where is the explicit any?

Reply:
the provider argument below is any, and this is part of the jest-pact library